docs/Reference/API-Methods.md

```diff
@@ -887,22 +887,25 @@ Returns the account balance of the specified address.
 
 **Returns**
 
-`result` : *QUANTITY* - Integer value of the current balance in wei.
+`result` : *QUANTITY* - Current balance in wei as a hexadecimal value.
+
+!!! important
```

Comment:
This note is correct but what it's saying applies to all examples.  I think we need to remove this one and add something general along these lines to the Attention box at the top of this page.  Something like: 

Except for the examples made on the Ropsten network, the example requests are made against private networks and your example results may be different depending on network configuration and activity.

Reply:
Updated.